### PR TITLE
Added static addresses and generate full user object

### PR DESCRIPTION
**Pull request by EndriFucia**

### CODE REVIEW

**Review by DrunkenToast — CHANGES_REQUESTED** [review]
No direct queries on the DB. Only API requests.

File: src/env/cities_postal_codes.ts [hunk 1]
```diff
@@ -0,0 +1,11032 @@
+  import { ICityPostCode } from '../models/address';
```
Comment (DrunkenToast) [inline src/env/cities_postal_codes.ts]: csv/json file instead?

**Comment by EndriFucia** [comment]
> No direct queries on the DB. Only API request.
Is there an api route ready for that?

**Comment by DrunkenToast** [comment]
> > No direct queries on the DB. Only API request.
> > Is there an api route ready for that?

You can check documentation:
https://github.com/tm-pe2/ad-api/blob/rewriteApi/docs/customers.md

If routes aren't ready you just prepare the objects for later.

**Review by DrunkenToast — CHANGES_REQUESTED** [review]

**Comment by DrunkenToast** [comment]
Also stop hashing, it gets done in the back-end => don't generate a hashed password.
Just send `password123` or similar.

File: src/env/cities_postal_codes.ts (hunk 1 above)
Comment (endrifucia1998) [reply src/env/cities_postal_codes.ts]: There is an array of cities under /env/ cities_postal_codes file which I use to insert them in the database

Comment (DrunkenToast) [reply src/env/cities_postal_codes.ts]: Yes my comment was to use csv instead and read them from file in code.

Comment (endrifucia1998) [reply src/env/cities_postal_codes.ts]: Is there a difference?

Comment (DrunkenToast) [reply src/env/cities_postal_codes.ts]: about 11k lines of differences (not loaded in RAM etc either)

Comment (DrunkenToast) [reply src/env/cities_postal_codes.ts]: but API calls are more important to me than this file.

Comment (DrunkenToast) [reply src/env/cities_postal_codes.ts]: https://github.com/jief/zipcode-belgium/blob/master/zipcode-belgium.json
Here is a json file you can use. This way you don't need to load it in unless needed from the json file.

Comment (endrifucia1998) [reply src/env/cities_postal_codes.ts]: https://github.com/spatie/belgian-cities-geocoded/blob/master/belgian-cities-geocoded.csv
I am on it with this one.

File: src/objects/address.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    for(let i = 0; i < data.length; i++)
+    {
+        const row = data[i].split(',');
+        const tempCity: ICityPostCode = {
+            postal_code: row[0],
+            city_name: row[1]
+        }
+        cities.push(tempCity);
+    }
+}
```
Comment (DrunkenToast) [inline src/objects/address.ts]: This parser might cause issues with splitting (for example line 390). Likely better to use a library for this. `csv-parse` is a library you can use. Alternatively load the json file instead.

**Review by DrunkenToast — CHANGES_REQUESTED** [review]

File: src/objects/address.ts (hunk 2 above)
Comment (DrunkenToast) [reply src/objects/address.ts]: Also my mistake, but why is this in ad-data and not in the SQL scripts?
I would assume this goes along with init sql script. We currently only have GET routes planned for the city routes.
Maybe there is an option for postgresql to insert csv with a script.

Comment (DriesOlbrechts) [reply src/objects/address.ts]: > Also my mistake, but why is this in ad-data and not in the SQL scripts? I would assume this goes along with init sql script. We currently only have GET routes planned for the city routes. Maybe there is an option for postgresql to insert csv with a script.

There is
```
COPY persons(first_name, last_name, dob, email)
FROM 'C:\sampledb\persons.csv'
DELIMITER ','
CSV HEADER;
```
source: https://www.postgresqltutorial.com/postgresql-tutorial/import-csv-file-into-posgresql-table/

Comment (DriesOlbrechts) [reply src/objects/address.ts]: will need a header however in the csv then, and multiple postal codes can't rly be allowed

Comment (EndriFucia) [reply src/objects/address.ts]: Ah perfect! Then I am removing all the addresses stuff from here.

**Review by DriesOlbrechts — APPROVED** [review]

**Pull request merged**